.github/workflows/s390x-build-binaries.yml

```diff
@@ -0,0 +1,50 @@
+name: s390x-build-binaries
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - nightly
+    tags:
+      # Final Release tags look like: v1.11.0
+      - v[0-9]+.[0-9]+.[0-9]+
+      # Release candidate tags look like: v1.11.0-rc1
+      - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+      - ciflow/nightly/*
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: [linux.s390x]
+    timeout-minutes: 720
```

Comment:
12 hours?!?! Does it really take that long? Can it reasonably limited to 4-5 hours?

Reply:
Yeah, it was taken from my CI workflow unchanged. Much lower value should be enough here since no tests are running.